central/telemetry/phonehome/gatherer_test.go

```diff
@@ -0,0 +1,81 @@
+package phonehome
+
+import (
+	"context"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stretchr/testify/suite"
+)
+
+type gathererTestSuite struct {
+	suite.Suite
+	mockCtrl *gomock.Controller
+}
+
+var _ interface {
+	suite.SetupTestSuite
+	suite.TearDownTestSuite
+} = (*gathererTestSuite)(nil)
+
+func (s *gathererTestSuite) SetupTest() {
+	s.mockCtrl = gomock.NewController(s.T())
+}
+
+func (s *gathererTestSuite) TearDownTest() {
+	s.mockCtrl.Finish()
+}
```

Comment:
No need to call [mockCtrl.Finish](https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish) if we passed `testing.T` to the `NewController` function.
You can remove this + the `suite.TearDownTestSuite` part.